ambry-api/src/main/java/com/github/ambry/router/GetBlobOptions.java

```diff
@@ -115,6 +119,10 @@ public boolean hasBlobSegmentIdx() {
     return blobSegmentIdx != NO_BLOB_SEGMENT_IDX_SPECIFIED;
   }
 
+  public RestRequest getRestRequest() {
```

Comment:
For consistency, let's just add a javadoc to this method, even if its somewhat redundant with the builder.